cli/templatepull_test.go

```diff
@@ -95,7 +95,7 @@ func TestTemplatePull_Stdout(t *testing.T) {
 
 	// Verify .zip format
 	tarReader := tar.NewReader(bytes.NewReader(expected))
-	expectedZip, err := coderd.CreateZipFromTar(tarReader)
+	expectedZip, err := fileszip.CreateZipFromTar(tarReader, int64(len(expected)))
```

Comment:
This seems potentially falky, should we really assume a zip will always be smaller than a tar?

Reply:
Let's export and use `coderd.HTTPFileMaxBytes` here as it's going to be the same limit.

Reply:
updated